sdk/impactreporting/Azure.ResourceManager.ImpactReporting/api/Azure.ResourceManager.ImpactReporting.net8.0.cs

```diff
@@ -0,0 +1,616 @@
+    {
+        public InsightProperties(string category, Azure.ResourceManager.ImpactReporting.Models.Content content, string insightUniqueId, Azure.ResourceManager.ImpactReporting.Models.ImpactDetails impact) { }
+        public Azure.ResourceManager.ImpactReporting.Models.InsightPropertiesAdditionalDetails AdditionalDetails { get { throw null; } set { } }
+        public string Category { get { throw null; } set { } }
+        public Azure.ResourceManager.ImpactReporting.Models.Content Content { get { throw null; } set { } }
+        public string EventId { get { throw null; } set { } }
+        public System.DateTimeOffset? EventOn { get { throw null; } set { } }
+        public string GroupId { get { throw null; } set { } }
+        public Azure.ResourceManager.ImpactReporting.Models.ImpactDetails Impact { get { throw null; } set { } }
+        public string InsightUniqueId { get { throw null; } set { } }
+        public Azure.ResourceManager.ImpactReporting.Models.ProvisioningState? ProvisioningState { get { throw null; } }
+        public string Status { get { throw null; } set { } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ImpactReporting.Models.InsightProperties System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ImpactReporting.Models.InsightProperties>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ImpactReporting.Models.InsightProperties>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ImpactReporting.Models.InsightProperties System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.InsightProperties>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.InsightProperties>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.InsightProperties>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class InsightPropertiesAdditionalDetails : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ImpactReporting.Models.InsightPropertiesAdditionalDetails>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.InsightPropertiesAdditionalDetails>
```

Comment:
do your spec have anonymous model? This name reads like an anonymous model.

And this name should also follow the change of model `InsightProperties`.